dev/com.ibm.ws.appclient.boot.cmdline/resources/com/ibm/ws/appclient/boot/resources/ClientLauncherOptions.nlsprops

```diff
@@ -98,8 +102,10 @@ option-desc.archive=\
 option-key.include=\
 \ \ \ \ --include=value,value,...
 option-desc.include=\
-\tA comma-delimited list of values. The valid values vary depending on \n\
-\tthe action.
+\tA comma-delimited list of values. The valid values are: all*, usr,   \n\
+\tminify. The usr option creates a package that contains the server,   \n\
+\tand application and not the runtime. The default all option produces \n\
```

Comment:
Hi, I made a similar suggestion in a different file. The comma placement was confusing me a little. Is this correct? 
+\tminify. The usr option creates a package that contains the server    \n\
+\tand application, but not the runtime. The default all option produces\n\
+\ta client package that contains everything.

If that last part needs to remain "server and application, and not the runtime" that should be fine too.